shipyard-templates/shipyard_templates/spreadsheets.py

```diff
@@ -10,7 +10,7 @@ class Spreadsheets(ABC):
     EXIT_CODE_DB_CREATE_ERROR = 204
     EXIT_CODE_BAD_REQUEST = 205
     EXIT_CODE_FILE_NOT_FOUND = 206
-    EXIT_CODE_UNKNOWN_ERROR = 299
+    EXIT_CODE_UNKNOWN_ERROR = 249
```

Comment:
was this intentional?

Reply:
This was intentional, there is a constraint in the application that doesn't allow for exit codes > 250